pkg/services/ngalert/api/api_alertmanager.go

```diff
@@ -10,7 +10,7 @@ import (
 	"time"
 
 	"github.com/go-openapi/strfmt"
-	"github.com/grafana/alerting/alerting"
+	"github.com/grafana/alerting/notify"
```

Comment:
How do you feel about using the prefix `alerting_` for all `grafana/alerting` packages?

Right now, the Alertmanager is _still_ imported directly in some areas (e.g. configuration), so it's a bit confusing to know if this is the upstream notify package or the alerting package.

While we do have plans to eventually remove the direct dependency of the Alertmanager until we do I say we avoid shooting ourselves in the foot and simply use `alerting_notify`, `alerting_receivers` et al.

Once the complete removal of the direct Alertmanager dependency is done - it'll be dead obvious that unless you see a modification to the `go.mod` you aren't importing something related to the Alertmanger directly and thus removes the need for this rename.

Reply:
I decided to use camel case as we already use `alertingModels` everywhere and GoLand's default style settings do not seem to like snake case.